File: uniapp_flutter/lib/models/splash_screen_properties.dart
class SplashScreenproperties {
  String splashicon;
  String splashduration;
  String splashbackground;
  String loginicon;

  SplashScreenproperties(
      {this.splashicon,
        this.splashduration,
        this.splashbackground,
        this.loginicon});

  SplashScreenproperties.fromJson(Map<String, dynamic> json) {
    splashicon = json['splashicon'];
    splashduration = json['splashduration'];
    splashbackground = json['splashbackground'];
    loginicon = json['loginicon'];
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['splashicon'] = this.splashicon;
    data['splashduration'] = this.splashduration;
    data['splashbackground'] = this.splashbackground;
    data['loginicon'] = this.loginicon;
    return data;
  }
}

File: app/app/src/main/java/com/vassar/unifiedapp/model/SplashScreenProperties.java
package com.vassar.unifiedapp.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
@JsonIgnoreProperties(ignoreUnknown = true)
public class SplashScreenProperties {
    @JsonProperty("splashduration")
    public String mSplashDuration;
    @JsonProperty("splashicon")
    public String mSplashIconUrl;
    @JsonProperty("loginicon")
    public String mLoginIcon;
    @JsonProperty("splashbackground")
    public String mSplashBackground;

    public SplashScreenProperties() {
    }

    public SplashScreenProperties(String splashDuration, String splashIconUrl
            , String loginIcon, String splashBackground) {
        this.mSplashDuration = splashDuration;
        this.mSplashIconUrl = splashIconUrl;
        this.mLoginIcon = loginIcon;
        this.mSplashBackground = splashBackground;
    }

    @Override
    public String toString() {
        return "SplashScreenProperties{" +
                "mSplashDuration='" + mSplashDuration + '\'' +
                ", mSplashIconUrl='" + mSplashIconUrl + '\'' +
                ", mLoginIcon='" + mLoginIcon + '\'' +
                ", mSplashBackground='" + mSplashBackground + '\'' +
                '}';
    }
}